packages/flutter/lib/src/painting/box_shadow.dart

```diff
@@ -36,11 +36,15 @@ class BoxShadow extends ui.Shadow {
     Offset offset = Offset.zero,
     double blurRadius = 0.0,
     this.spreadRadius = 0.0,
+    this.blurStyle = BlurStyle.normal,
   }) : super(color: color, offset: offset, blurRadius: blurRadius);
 
   /// The amount the box should be inflated prior to applying the blur.
   final double spreadRadius;
 
+  /// The style(inner/normal/outer/solid) of the blur.
```

Comment:
Let's keep the formatting/doc style consistent with the rest of Flutter. How about:

```
/// The [BlurStyle] to use for this shadow.
///
/// Defaults to [BlurStyle.normal].
```

Reply:
Sure thing @GaryQian